### ISSUE TITLE
Trace Back Error After  A Few Seconds

### ISSUE THREAD

**Issue by pierrechn** [instabot-py/instabot.py#2067]
Hello,

I continue to get this error message every time, are there any known solutions?

![Screenshot (527)](https://user-images.githubusercontent.com/43892593/55197295-203c5780-5180-11e9-94cb-727adc5b2d52.png)

**Comment by dimisdas** [comment]
1. The issue template is missing from your post

2. You are using an older version. Please open issues if you’re having problems with the current version. 

Thank you

**Comment by pierrechn** [comment]
@dimisdas older version? what are you referring to and how can I update it? it's the most recent version of python.

**Comment by dimisdas** [comment]
Take a look at README.md.

**Comment by pierrechn** [comment]
@dimisdas just downloaded it, how do i access the file? it's an .md extention on my windows.

**Comment by pierrechn** [comment]
@dimisdas just pulled it up, the readme isn't really specific.

I downloaded the latest 'instabot.py' setup zip file, the pip and all that stuff was previously installed.

What do you suggest I do now?

**Comment by dimisdas** [comment]
Which part is not specific?

**Comment by pierrechn** [comment]
You said my issue was that Instabot was not updated or at least this is what was insinuated. The readme file doesn't mention how to update the program. I used TextEdit to open the document so maybe it messed up the format. 

Can you copy and past the directions to this thread?

Can you provide a walk through on how to update the program to see if that solves the problem? It worked fine up until a few days ago.

I have been getting this syntax error trace for the last few days.

**Comment by dimisdas** [comment]
Considering you were running the bot on a very old version, you’ll have to reinstall.

Which means you should follow the installation steps. The project is available on PyPi, so the “pip install instabot-py” command should install everything. You don’t need to download a .zip or a file.

After installing with pip, you may run the bot with the command “instabot-py”.

**Comment by pierrechn** [comment]
@dimisdas can you provide a link to the correct pip install ? Not locating it.

**Comment by pierrechn** [comment]
@dimisdas I'm not very experienced with this type of software so it's very confusing. I found a tutorial video for that old version a few months back that broke everything down.

**Comment by dimisdas** [comment]
That's okay. I'll help you. You should however put an effort when asking for help. When you created this issue here, there were several questions with commands you should run, and questions about your environment. You should have read them carefully and run them, because now I have no idea what version you're running (python and pip).

You said you are running` the most recent version of python`, however I can see in your screenshot it's 3.7.1rc (current is 3.7.3).

I asked you to install instabot-py with a specific command "`pip install instabot-py`". Did this command run succesfully? Was there an error? No clue! How am I supposed to help?

And now you're writing `can you provide a link to the correct pip install ? Not locating it.`. No idea what this means. You don't know how to run pip? Or you don't have pip installed?

I need you to be very exact with the steps you're taking, as well as describe them. Always include **all relevant output** from the commands. `Not locating it` is not something I can help you with, you need to be specific.

[Take a look at this helpful guide on how to ask questions the smart way.](http://www.catb.org/~esr/faqs/smart-questions.html) It's not just about being polite and saving other people's time, but most importantly not wasting yours :)

Don't take this the bad way.

**Comment by pierrechn** [comment]
No worries, I just came across the originally video and used it to gain access in Terminal once again. I'm upgrading everything now based on what you're saying combined with a tutorial on how to work the terminal app.

PIP 19.0.3 is installed along with Python 3.7.3

Here's my current issue, checkout the screen shot.

![Screenshot (528)](https://user-images.githubusercontent.com/43892593/55247575-0188a000-5216-11e9-905e-f7efa4c7c2e3.png)

**Comment by dimisdas** [comment]
I see. You're still running the old version in the screenshot.

Search and run CMD (command line). You should run the commands

- pip --version
and
- pip3.7 --version

In case multiple version are installed. Please report with the outputs.

**Comment by pierrechn** [comment]
Interesting, if I go by these two then everything appears to be installed correctly.

![Screenshot (529)](https://user-images.githubusercontent.com/43892593/55247974-e4080600-5216-11e9-9ca4-02ef910569bc.png)
![Screenshot (530)](https://user-images.githubusercontent.com/43892593/55248064-0732b580-5217-11e9-83d4-d8dd492b3372.png)

**Comment by dimisdas** [comment]
Nice. Now in the same CMD prompt run `instabot-py`

**Comment by pierrechn** [comment]
Okay now I'm getting this, should I download the instabot folder again?

Also the .py files are no longer showing the IDLE symbol, they're just blank white spaces now. Does this mean something was uninstalled?

![Screenshot (531)](https://user-images.githubusercontent.com/43892593/55248349-9d66db80-5217-11e9-963e-c5fb4a5c50af.png)
![Screenshot (532)](https://user-images.githubusercontent.com/43892593/55248350-9d66db80-5217-11e9-83c4-5c60a4943cd8.png)

**Comment by dimisdas** [comment]
The command is `instabot-py`, not `instabot.py`

**Comment by pierrechn** [comment]
I see, I'm still getting an error. I'm going to re-download the full folder and try the command that way and see if anything works.

![Screenshot (533)](https://user-images.githubusercontent.com/43892593/55248839-a310f100-5218-11e9-866c-82698a795d54.png)

**Comment by dimisdas** [comment]
I'm sorry, but I just can't help you this way. No patience, no following instructions. I even specificaly said you don't need to download the folder. I've tried my best.

Good luck and goodbye.

**Comment by pierrechn** [comment]
I've told you multiple times that I'm not familiar with Python or the Terminal. (or any of this stuff tbh)

I need you to be very specific on what you mean when you give an instruction. You are obviously a very experienced user, so you have to keep that in mind when communicating. 

I've literally never used terminal until I downloaded this app.

**Comment by pierrechn** [comment]
you all should have a featured instructional video available on this website to walk users through the process. that would negate majority of these threads that i've read on here.

**Comment by pierrechn** [comment]
Hey,

I just wanted to let you know that I was able to get the bot up and running but I found a new issue. I'm currently going through issues to locate what's going on. It took a few days but after a few tries I was able to set it back up. 

The black screen is what happens when i run 'Instabot.py' on CMD

and the blue screen error is what I get when running it on IDLE 3.7. (which is how I originally used the old version of the bot)

![Screenshot (537)](https://user-images.githubusercontent.com/43892593/55375657-a46d4280-54d2-11e9-9891-9fe2dcaceb61.png)

![Screenshot (539)](https://user-images.githubusercontent.com/43892593/55375732-de3e4900-54d2-11e9-9828-83ef12463dcf.png)
